machine/delta/delta_ag9064/i2ctools/i2ctools-sys-eeprom-platform-2.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/sys_eeprom/24cXX.c b/sys_eeprom/24cXX.c
```

Comment:
For a couple of years now, the ONIE project does not accept the "i2ctools" implementation of onie-syseeprom for new platforms. We have standardized on the busybox implementation. Discussed here:

http://lists.opencompute.org/pipermail/opencompute-onie/2015-March/000512.html

The busybox implementation is much better, more configurable without the need for platform specific patches to change things like the delay timing. See this example:
https://github.com/opencomputeproject/onie/blob/master/machine/accton/accton_as5812_54x/busybox/conf/config

Also see these commits that added the busybox implementation:
fc76004
481b975

Reply:
OK